pilot/cmd/pilot-agent/main.go

```diff
@@ -99,6 +99,11 @@ var (
 		"the provider of Pilot DNS certificate.").Get()
 	jwtPolicy = env.RegisterStringVar("JWT_POLICY", jwt.PolicyThirdParty,
 		"The JWT validation policy.")
+	// ProvCert is the environment controlling the use of pre-provisioned certs, for VMs.
+	// May also be used in K8S to use a Secret to bootstrap (as a 'refresh key'), but use short-lived tokens
+	// with extra SAN (labels, etc) in data path.
+	provCert = env.RegisterStringVar("PROV_CERT", "",
```

Comment:
Isn't this duplicate with https://github.com/istio/istio/blob/3e370494e25e3454ef9864793408ca5f75684a29/security/pkg/nodeagent/caclient/providers/citadel/client.go#L47

Reply:
This is inserted into the security options

Reply:
It is required for the main.go to transmit the options to security.Options

Reply:
I assume this is a cleanup - and you deleted the one in citadel/client.go ?